### PR TITLE
[WEB-1643,WEB-1670] Responsive styling updates for the Clinic UI

### PR DESCRIPTION
**Pull request by clintonium-119**
See [WEB-1643] and [WEB-1670]

[WEB-1643]: https://tidepool.atlassian.net/browse/WEB-1643?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WEB-1670]: https://tidepool.atlassian.net/browse/WEB-1670?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/themes/base/containers.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
 export default ({ borders, colors, radii, space }) => {
   const defaultStyles = {
-    mx: 'auto',
+    mx: [0, 'auto'],
     bg: colors.white,
-    width: ['100%', '85%'],
+    width: ['100%', '95%'],
+    // width: auto,
```
Comment (krystophv) [inline app/themes/base/containers.js:6]: One more comment hanging out

File: app/themes/base/tables.js [hunk 2]
```diff
@@ -53,6 +59,81 @@ export default ({ borders, colors, fonts, fontSizes, shadows }) => {
...
+      display: ['block', null, 'table-row'],
+      position: ['relative', null, 'static'],
+
+
+      // display: ['flex', null, 'table-row'],
+      // justifyContent: 'space-between',
+      // flexWrap: 'wrap',
+      // columnGap: '24px',
+      // rowGap: '8px',
+      // alignItems: 'center',
```
Comment (krystophv) [inline app/themes/base/tables.js:98]: Comment block seems leftover

**Review by krystophv — APPROVED** [review]
LGTM :shipit: 

A handful of commented code blocks that can likely be trimmed out, but nothing worth holding up approval for.

File: app/pages/clinicadmin/clinicadmin.js [hunk 3]
```diff
@@ -480,17 +498,18 @@ export const ClinicAdmin = (props) => {
...
-        field: 'edit',
-        render: renderEdit,
-        align: 'left',
-      },
+      // {
+      //   title: '',
+      //   field: 'edit',
+      //   render: renderEdit,
+      //   align: 'left',
+      // },
```
[the remark this answers is not in the document]
Comment (clintonium-119) [reply app/pages/clinicadmin/clinicadmin.js]: Removed :+1:

File: app/pages/clinicworkspace/ClinicPatients.js [hunk 4]
```diff
@@ -1516,6 +1553,7 @@ export const ClinicPatients = (props) => {
 
         {pageCount > 1 && (
           <Pagination
+            // variant={['condensed', 'default']}
```
[the remark this answers is not in the document]
Comment (clintonium-119) [reply app/pages/clinicworkspace/ClinicPatients.js]: Removed :+1:

File: app/themes/base/containers.js (hunk 1 above)
Comment (clintonium-119) [reply app/themes/base/containers.js]: Removed :+1:

File: app/themes/base/tables.js (hunk 2 above)
Comment (clintonium-119) [reply app/themes/base/tables.js]: Removed :+1:

**Pull request merged**